app/views/candidate_interface/decisions/offer.html.erb

```diff
@@ -15,16 +15,13 @@
 
       <%= render(CandidateInterface::OfferReviewComponent.new(course_choice: @application_choice)) %>
 
-      <p class="govuk-body">Before you accept:</p>
-
-      <ul class="govuk-list govuk-list--bullet">
-        <% if @offer_count > 1 %>
-          <li>if you accept this offer, your other <%= 'offer'.pluralize(@offer_count - 1) %> will be automatically declined</li>
-        <% end %>
-        <li>if you decline all of your offers, or do not respond in time, you can still apply for courses that start this academic year</li>
-      </ul>
+      <% if @offer_count > 1 %>
+         <p class="govuk-body">If you accept this offer, your other <%= 'offer'.pluralize(@offer_count - 1) %> will be automatically declined.</p>
+      <% end %>
+        
+      <p class="govuk-body">If you decline your <%= 'offer'.pluralize(@offer_count - 1) %>, or do not respond in time, you can still apply for courses that start this academic year.</li>
```

Comment:
without any end of cycle logic in here, is 'you can still apply for courses that start this academic year' always true?